### PR TITLE
WWW-404 Overriding .emailSubscription-wrapper in locations app.

### PR DESCRIPTION
**Pull request by holingpoon**
This PR should override the box width for .emailSubscription-wrapper on locations.

Please also see Jira comment for struck-out text issue. Thanks!

https://jira.nypl.org/browse/WWW-404?focusedCommentId=26272&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-26272

### CODE REVIEW

File: public/css/partials/_global.scss [hunk 1]
```diff
@@ -57,6 +57,14 @@ input[type="text"] {
   box-sizing: content-box;
 }
 
+.emailSubscription-wrapper {
```
Comment (ktp242) [inline public/css/partials/_global.scss]: I think we can delete line 52 to line 58. They are the same styles to the same element, but just different letter cases.

File: public/css/partials/_global.scss [hunk 2]
```diff
@@ -49,7 +49,7 @@ input[type="text"] {
 }
 
 // To make the width of the subscribe container wider.
-.EmailSubscription-Wrapper {
+.emailSubscription-wrapper {
```
Comment (EdwinGuzman) [inline public/css/partials/_global.scss]: Wow, we had a typo here?! :(

**Comment by ktp242** [comment]
We have conflicts for min.css files. Pull down development, compile again, and re push should be able to fix it.

**Comment by holingpoon** [comment]
Repushed after merge with development and recompilation.

File: public/css/partials/_global.scss (hunk 2 above)
Comment (holingpoon) [reply public/css/partials/_global.scss]: I think the older version of the CSS selector was capitalized, then somehow the naming convention changed to this 🤔 That was the name format I got from Chrome Inspector.

File: locinator.json [hunk 3]
```diff
@@ -15,7 +15,7 @@
     },
     "qa": {
       "url": "http://qa.locations.nypl.org",
-      "api": "https://refinery.nypl.org/api/nypl/locations",
+      "api": "https://qa-refinery.nypl.org/api/nypl/locations",
```
Comment (ktp242) [inline locinator.json]: I am not sure qa refinery is currently working

**Pull request merged**